indexer/httptransport.go

```diff
@@ -84,7 +102,14 @@ func (h *HTTP) IndexReportHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+const (
+	linkIndex  = `<%s>; rel="https://projectquay.io/clair/v1/index_report"`
```

Comment:
hardcoded urls? Clair will be used outside of quay.

Reply:
The URI describes the relation, and per the RFC must be an absolute URI.
The link target (bit inside the angles) is completely relative.

So, in effect this is just denoting what is found at the link target.